server/routes/column.js

```diff
@@ -0,0 +1,20 @@
+const express = require("express");
+const router = express.Router();
+const protect = require("../middleware/auth");
+const {
+  getColumns,
+  createColumn,
+  getColumn,
+  updateColumn,
+  reorderColumn,
+} = require("../controllers/column");
+
+router.use(protect);
+
+router.route("/").get(getColumns);
+router.route("/").post(createColumn);
+router.route("/:id").get(getColumn);
+router.route("/:id").patch(updateColumn);
+router.route("/:id").patch(reorderColumn);
```

Comment:
To express the two patch requests are identical, as far as I know express doesn't differentiate by different methods in the patch requests. So the `reorder` patch would need to be different to be callable. I would just do `/:id/reorder`. 

Great use of using patch here too!

Reply:
Exactly. There'll need to be either 2 different routes or 1 function that will differentiate within it how to update